components/style/values/specified/box.rs

```diff
@@ -130,3 +131,63 @@ define_css_keyword_enum! { OverflowClipBox:
     "content-box" => ContentBox,
 }
 add_impls_for_keyword_enum!(OverflowClipBox);
+
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue)]
+/// Provides a rendering hint to the user agent,
+/// stating what kinds of changes the author expects
+/// to perform on the element
+///
+/// <https://drafts.csswg.org/css-will-change/#will-change>
+pub enum WillChange {
+    /// Expresses no particular intent
+    Auto,
+    /// scroll-position | contents | <custom-ident>
```

Comment:
You can `derive(ToCss)` and use `#[css(iterable, comma)]` in this variant.

I'd remove the `scroll-position | contents` productions since we don't use them, but it's not a big deal anyway.

Reply:
Let me try to use `derive(ToCss)` for it! Thanks!


> I'd remove the scroll-position | contents productions since we don't use them
Got it. I add them after reading the spec but I think it might be better to remove them due to not using anymore.